modules/gcp-gke/network.tf

```diff
@@ -81,3 +80,18 @@ resource "google_compute_firewall" "gke_private_cluster_public_https_firewall_ru
   source_ranges = ["0.0.0.0/0"]
   target_tags   = [local.gke_node_pool_tag]
 }
+
+resource "google_compute_firewall" "gke_private_cluster_allow_9443_kyverno" {
```

Comment:
As discussed in standup, I think we should refactor `gke_private_cluster_istio_gatekeeper_rules` into a generic option where we can open any number of ports by just adding them to the inputs of this module.  Otherwise every time we need to come back here and update this module too.

Reply:
Done @MXfive